Description: Some Canonical OEM partners have requested additional options
  be added to the power option system sleep settings menu.

  Add 20 minutes and 2 hours options to give the user
  more fine-grained controls. (LP: #1244065)
Author: Jian-Ding Chen (timchen119) <tim.chen119@canonical.com>
Last-Update: 2013-10-24
Index: gnome-control-center/panels/power/power.ui
===================================================================
--- gnome-control-center.orig/panels/power/power.ui	2013-09-27 14:24:32.016344000 +0800
+++ gnome-control-center/panels/power/power.ui	2013-09-27 14:25:04.799258891 +0800
@@ -40,6 +40,10 @@
         <col id="1">600</col>
       </row>
       <row>
+        <col id="0" translatable="yes">20 minutes</col>
+        <col id="1">1200</col>
+      </row>
+      <row>
         <col id="0" translatable="yes">30 minutes</col>
         <col id="1">1800</col>
       </row>
@@ -48,6 +52,10 @@
         <col id="1">3600</col>
       </row>
       <row>
+        <col id="0" translatable="yes">2 hours</col>
+        <col id="1">7200</col>
+      </row>
+      <row>
         <col id="0" translatable="yes">Don't suspend</col>
         <col id="1">0</col>
       </row>
